src/Servers/Kestrel/Core/src/Internal/Http2/Http2Stream.FeatureCollection.cs

```diff
@@ -30,5 +34,19 @@ IHeaderDictionary IHttpResponseTrailersFeature.Trailers
         }
 
         int IHttp2StreamIdFeature.StreamId => _context.StreamId;
+
+        MinDataRate IHttpMinRequestBodyDataRateFeature.MinDataRate
+        {
+            get => throw new NotSupportedException(CoreStrings.Http2MinDataRateNotSupported);
```

Comment:
The getter should return null if that's the only allowed value.

Reply:
The thing is we generally are enforcing a rate limit for HTTP/2, it's just per connection, not per stream. So I don't think it makes sense to return null, but I don't think it makes sense to return the connection limit either.